<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cameraWindow</class>
 <widget class="QMainWindow" name="cameraWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>957</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cameraWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Camera_On_Button">
    <property name="geometry">
     <rect>
      <x>921</x>
      <y>179</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开相机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="video_On_Button">
    <property name="geometry">
     <rect>
      <x>921</x>
      <y>338</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Process_Button_3">
    <property name="geometry">
     <rect>
      <x>921</x>
      <y>391</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>图片处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartRec_Button">
    <property name="geometry">
     <rect>
      <x>921</x>
      <y>444</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始录制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EndRec_Button">
    <property name="geometry">
     <rect>
      <x>921</x>
      <y>497</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>结束录制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseCamera_Button">
    <property name="geometry">
     <rect>
      <x>921</x>
      <y>285</y>
      <width>97</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭相机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EndRec_Button_2">
    <property name="geometry">
     <rect>
      <x>921</x>
      <y>232</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>拍照</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveScrpic_Button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>921</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存照片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Quit_Button">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>630</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cali_Button">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>标定</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>891</width>
      <height>893</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>C:/Users/ssrzm/Desktop/搞笑图片/006AyCZZly1fk9s1ymsqrj30hs0hswfn.jpg</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>C:/Users/ssrzm/Desktop/搞笑图片/006AyCZZly1fk9s1ymsqrj30hs0hswfn.jpg</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>C:/Users/ssrzm/Desktop/搞笑图片/006AyCZZly1fk9s1ymsqrj30hs0hswfn.jpg</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>C:/Users/ssrzm/Desktop/搞笑图片/006AyCZZly1fk9s1ymsqrj30hs0hswfn.jpg</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Cali_Button_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>双标定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process_Button">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>590</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>深入处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process_Button_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>660</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>深入处理</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
